eng/Versions.props

```diff
@@ -15,10 +15,9 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
   <PropertyGroup>
-    <VersionPrefix>16.6.0</VersionPrefix>
+    <VersionPrefix>16.6.1</VersionPrefix>
```

Comment:
@rainersigwald isn't this technically 16.6.2 since June would have been 16.6.1 of VS?

Reply:
We usually diverge our versions from VS: VS 16.5.5 shipped MSBuild 16.5.1, for instance.